src/scene/data.ts

```diff
@@ -248,7 +250,7 @@ export async function exportCanvas(
 
 function restore(
   savedElements: readonly ExcalidrawElement[],
-  savedState: AppState,
+  savedState: AppState | null,
```

Comment:
Why do you have to make this nullable?

Reply:
The problem here is that, in line 286-296 in `restoreFromLocalStorage` function, the app state can be null (i.e if JSON parsing fails). When it fails, it would just set the `appState` to null. If you have an idea for stopping this, I am open to suggestions but I do not know what else to do here.